src/Saritasa.Tools.EntityFrameworkCore/Saritasa.Tools.EntityFrameworkCore.csproj

```diff
@@ -19,8 +19,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1" PrivateAssets="All" />
```

Comment:
Why is it not needed?

Reply:
[Microsoft.SourceLink.GitHub"](https://github.com/dotnet/sourcelink?tab=readme-ov-file#githubcom-and-github-enterprise)
> Source Link package is a development dependency, which means it is only used during build. It is therefore recommended to set PrivateAssets to all on the package reference. This prevents consuming projects of your nuget package from attempting to install Source Link.

> Referencing any Source Link package in a .NET SDK 8+ project suppresses Source Link that is included in the SDK.

Seems it is not needed in .NET 8?